### DISCUSSION TITLE
Why a cannot change color of theme?!?!?

### DISCUSSION THREAD

**Discussion by shtapsel** [darkreader/darkreader#13117] [Q&A]
I create new browser profile and would custom reader. 
In my old profile it first that i see. But in new i cant find this points    

![Screenshot from 2024-08-10 22-46-37](https://github.com/user-attachments/assets/7a590fad-4a0a-43fa-8b65-bf2bdce38720)

**Comment by Obsidianninja11** [comment]
Go to Settings > Advanced > Dev Tools > Advanced > Switch to old design

<details><summary>Images</summary>

![image](https://github.com/user-attachments/assets/5fc08710-78f6-45db-98a8-fc54556f4d7e)
![image](https://github.com/user-attachments/assets/772f3f80-f704-4c97-b9b0-2791fb572bed)
![image](https://github.com/user-attachments/assets/c9697b94-f1c0-4f6a-a7a1-9d2843da3abb)
</details>

**Comment by shtapsel** [comment]
Thanks
